sidebars.js

```diff
@@ -704,6 +704,7 @@ module.exports = {
               type: "category",
               label: "Zowe API Mediation Layer Security",
               items: [
+                "extend/extend-apiml/api-medation-sso-integration-extenders",
```

Comment:
It would seem logical that the API ML security overview should come before api-mediation-sso-integration-extenders.

Is there a reason why it this file is in the first position?

Reply:
Because the security overview, doesn't give you the security overview and will be replaced as the work on the improvement of extending section will progress.